### PR TITLE
Psycrush and psyblast multitile walance changes

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Reduces the AOE damage of Psyblast/crush against multitile vehicles.

Also removed some caste define stuff for warlock abilities. With the removal of maturity, they serve no purpose. This probably applies to the majority of ability caste vars, but we'll leave that for another pr.
## Why It's Good For The Game
Multitile vehicles are particularly vulnerable to any kind of AOE damage since it can hit up to 9 times. While I like the idea of being able to get better hits in, currently this means warlock in particular can do HUGE damage to vehicles.
## Changelog
:cl:
balance: Reduced AOE damage of psyblast/psycrush against multitile vehicles
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
its so over , we only have primo rock-slammer to insta-kill tanks now..

**Comment by blackdav123** [comment]
should probably include the shield changes in changelog

File: code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm [hunk 1]
```diff
@@ -399,13 +399,16 @@
...
+				carbon_victim.apply_damage(PSY_CRUSH_DAMAGE * 1.5, STAMINA, blocked = BOMB)
 				carbon_victim.adjust_stagger(5 SECONDS)
 				carbon_victim.add_slowdown(6)
 			else if(isvehicle(victim))
 				var/obj/vehicle/veh_victim = victim
-				veh_victim.take_damage(xeno_owner.xeno_caste.crush_strength * 5, BRUTE, BOMB)
+				var/dam_mult = 1.5 //multitile vehicles can be hit multiple times
+				if(ismecha(veh_victim))
+					dam_mult = 5
+				veh_victim.take_damage(PSY_CRUSH_DAMAGE * dam_mult, BRUTE, BOMB)
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm:411]: This ignores things like bikes? should this comment say stuff like mech only takes more dmg?

Comment (Lumipharon) [reply code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm:411]: true, eh I'll just remove the comment

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**